oh_staff_ui/management/commands/import_projectitems.py

```diff
@@ -0,0 +1,101 @@
+from django.core.management.base import BaseCommand
+from oh_staff_ui.models import ProjectItem, ItemStatus, ItemType
+from django.contrib.auth.models import User
+from csv import DictReader
+import os
+import datetime
+
+
+class Command(BaseCommand):
+    help = "Deletes all existing ProjectItem data, then imports from a CSV file"
+
+    def add_arguments(self, parser):
+        parser.add_argument("filename", type=str)
+
+    def handle(self, *args, **options):
+        # to avoid duplication, start by deleting all existing ProjectItems
+        print("Deleting all existing ProjectItems.")
+        ProjectItem.objects.all().delete()
+
+        # open and read input CSV - assuming it's in the fixtures folder
+        filepath = os.path.join("oh_staff_ui/fixtures", options["filename"])
+        with open(filepath, mode="r") as f:
+            dict_reader = DictReader(f)
+            projectitem_dicts = list(dict_reader)
+        total_rows = len(projectitem_dicts)
+        print(f"Found {total_rows} ProjectItems to import.")
+
+        # start with known top-level ProjectItem
+        # handled outside of main loop due to lack of parent
+        added_items = []
+        root_item = next(
+            item for item in projectitem_dicts if item["ARK"] == "21198/zz0008zh0f"
+        )
+        # create and save ProjectItem for root object
+        p = ProjectItem(
```

Comment:
Per my previous comment, this may not be necessary.  If it is still needed, I think it (and the `save()` at line 46) could be replaced with a call to `add_projectitem(root_item)`, though you may need to add a (blank/null) `parent` key first.